test/images/clusterapi-tester/Makefile

```diff
@@ -12,20 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: push
+BIN=clusterapi-tester
+ARCH ?= amd64
+TARGET ?= $(CURDIR)
+GOLANG_VERSION ?= latest
 
-# 0.0 shouldn't clobber any released builds
-TAG = 1.0
-PREFIX = gcr.io/google_containers/clusterapi-tester
+bin:
+	docker run -it -v $(TARGET):$(TARGET):Z -v `pwd`/../../../:/go/src/k8s.io/kubernetes:Z \
```

Comment:
In the past I faced some issue while building with volume mount on selinux enabled systems. This will help me overcome with that issue..